### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by thandayuthapani**
Helm can be installed without cloning code too,
Have added steps in README to install volcano using helm without cloning code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,8 +53,27 @@ the open source community.
...
+Add helm repo using following command,
+
+```
+helm repo add volcano https://volcano-sh.github.io/charts
+```
+
+Install Volcano using following command,
+
+```
+helm install volcano/volcano -namespace <namespace> --name <specified-name>
```
Comment (TommyLike) [inline README.md]: --namespace

File: README.md [hunk 2]
```diff
@@ -53,8 +53,27 @@ the open source community.
 
 ## Quick Start Guide
 
-The easiest way to deploy Volcano is to use the Helm chart.
+The easiest way to deploy Volcano is to use the Helm chart.  Volcano can be deployed by cloning code and also by adding helm repo.
 
+## Using Volcano Helm Repo
+## Helm Charts
```
Comment (TommyLike) [inline README.md]: looks like an incorrect level or redundant header
```
## Helm Charts
```

**Comment by TommyLike** [comment]
Now we have two different repos which both keep the volcano charts codes?

File: README.md (hunk 2 above)
Comment (thandayuthapani) [reply README.md]: Have removed that header

File: README.md (hunk 1 above)
Comment (thandayuthapani) [reply README.md]: changed to --namespace

**Comment by thandayuthapani** [comment]
> Now we have two different repos which both keep the volcano charts codes?

Currently we have it in volcano-sh/volcano repo too, once decision on whether to remove or keep it made, we can update README accordingly.

**Pull request merged**